Earthfile

```
@@ -0,0 +1,216 @@
+        tar xzf mysql-connector-odbc-5.1.13-linux-glibc2.5-x86-64bit.tar.gz && \
+        mysql-connector-odbc-5.1.13-linux-glibc2.5-x86-64bit/bin/myodbc-installer -d -a -n MySQL -t DRIVER=`ls /opt/mysql-*/lib/libmyodbc*w.so` && \
+        rm /opt/*.tar.gz
+
+    # we need an mssql odbc driver for the integration tests
+    RUN wget -q https://packages.microsoft.com/keys/microsoft.asc -O- | apt-key add - && \
+        wget -q https://packages.microsoft.com/config/debian/9/prod.list -O- > /etc/apt/sources.list.d/mssql-release.list && \
+        apt-get update && \
+        ACCEPT_EULA=Y apt-get install msodbcsql17 mssql-tools && \
+        odbcinst -i -d -f /opt/microsoft/msodbcsql17/etc/odbcinst.ini
+
+    # not used for the moment as Exasol is not tested here
+    # RUN cd /opt && \
+    #     wget -q https://www.exasol.com/support/secure/attachment/111075/EXASOL_ODBC-6.2.9.tar.gz && \
+    #     tar xzf EXASOL_ODBC-6.2.9.tar.gz && \
+    #     echo "\n[EXASOL]\nDriver=`ls /opt/EXA*/lib/linux/x86_64/libexaodbc-uo2214lv1.so`\nThreading=2\n" >> /etc/odbcinst.ini && \
+    #     rm /opt/*.tar.gz
+
+python:
+    ARG PYTHON_VERSION=3.8.6
+    ARG ARROW_VERSION_RULE="<2.0.0"
+    ARG NUMPY_VERSION_RULE=""
+    ARG CONDA_EXTRA=""
+    FROM +os-base
+
+    RUN mamba create -y -q -n turbodbc-dev \
+        c-compiler \
+        make \
+        ninja \
+        cmake \
+        coveralls \
+        gmock \
+        gtest \
+        cxx-compiler \
+        mock \
+        pytest \
+        pytest-cov \
+        python=${PYTHON_VERSION} \
+        unixodbc \
+        boost-cpp \
+        numpy$NUMPY_VERSION_RULE \
+        pyarrow$ARROW_VERSION_RULE \
+        pybind11 \
+        $CONDA_EXTRA
+
+    RUN echo "conda activate turbodbc-dev" >> ~/.bashrc
+    RUN echo 'export UNIXODBC_INCLUDE_DIR=$CONDA_PREFIX/include' >> ~/.bashrc
+
+    RUN cd /opt && \
+        wget -q https://github.com/pybind/pybind11/archive/v2.6.2.tar.gz && \
+        tar xvf v2.6.2.tar.gz
+
+build:
+    ARG PYTHON_VERSION=3.8.6
+    ARG ARROW_VERSION_RULE="<2.0.0"
+    ARG NUMPY_VERSION_RULE=""
+    ARG CONDA_EXTRA=""
+    FROM --build-arg PYTHON_VERSION="$PYTHON_VERSION" \
+        --build-arg ARROW_VERSION_RULE="$ARROW_VERSION_RULE" \
+        --build-arg NUMPY_VERSION_RULE="$NUMPY_VERSION_RULE" \
+        --build-arg CONDA_EXTRA="$CONDA_EXTRA" \
+        +python
+
+    COPY . /src
+    # remove a potential available host build
+    RUN rm -rf /src/build && mkdir /src/build
+    WORKDIR /src/build
+
+    ENV ODBCSYSINI=/src/earthly/odbc
+    ENV TURBODBC_TEST_CONFIGURATION_FILES="query_fixtures_postgresql.json,query_fixtures_mssql.json,query_fixtures_mysql.json"
+    RUN ln -s /opt/pybind11-2.6.2 /src/pybind11
+
+    RUN bash -ic " \
+        cmake -DBOOST_ROOT=$CONDA_PREFIX -DBUILD_COVERAGE=ON \
+            -DCMAKE_INSTALL_PREFIX=./dist  \
+            -DPYTHON_EXECUTABLE=`which python` \
+            -GNinja .. && \
```

You need to add the correct escaping as otherwise the shell commands are already executed in the outer bash context. 
```suggestion
        cmake -DBOOST_ROOT=\${CONDA_PREFIX} -DBUILD_COVERAGE=ON \
            -DCMAKE_INSTALL_PREFIX=./dist  \
            -DPYTHON_EXECUTABLE=\$(which python) \
            -GNinja .. && \
```